src/logic/calculate.test.js

```diff
@@ -1,6 +1,16 @@
 import calculate from './calculate';
 
-it('should return an object', () => {
-  const calculateTest = calculate({ total: 0, next: null, operation: null }, 'AC');
-  expect(calculateTest).toStrictEqual({ total: 0, next: null, operation: null });
+describe('calculate', () => {
+  it('should return the sum of two numbers', () => {
+    const result = calculate({
+      total: 0,
+      next: null,
+      operation: null,
+    }, 'AC');
+    expect(result).toEqual({
+      total: 0,
+      next: null,
+      operation: null,
+    });
+  });
 });
```

Comment:
- [ ] You are just testing your AC button. Please add unit tests in which total, next, operation values will change. :blush: as previous reviewer asked you to do it. If it is difficult for you to understand then reach out to me on slack by searching my name **Hammas** in **full time student** channel :+1:

Reply:
Thank you so much for your time @HAMMAS-SALEEM